nav2_controller/include/nav2_controller/nav2_controller.hpp

```diff
@@ -34,36 +34,114 @@ namespace nav2_controller
 {
 
 class ProgressChecker;
-
+/**
+ * @class nav2_controller::ControllerServer
+ * @brief This class publishes twist velocity for robot by computing velocities
```

Comment:
Should I use DWB planner instead?

Reply:
or local controller?

Reply:
And also, I think it uses only local controller plugin. I don't see any other plugins.

Reply:
just use `controller`, a controller by definition is going to be local.

It will use multiple plugins very very shortly, I think its safe to add that to the docs

Reply:
What do you think?
```suggestion
 /**
 * @class nav2_controller::ControllerServer
 * @brief This class hosts variety of plugins of different algorithms to
 * complete control tasks. 
 *
 * This class hosts FollowPath action server which receives path from action
 * client and it is feed to controller which calculates and publishes twist
 * velocities.
 */
```

Reply:
> This class hosts variety of plugins of different algorithms to complete control tasks from the exposed FollowPath action server.

Reply:
I think the detailed description can updated when we know which different plugins to be used in it.

Reply:
The point of the server is to host any arbitrary controller plugin, so there shouldn't really be any specifics about the task that its trying to accomplish given the algorithm unkown

Reply:
I removed "hosts" from the brief as it will be obvious for server to host
```
/**
 * @class nav2_controller::ControllerServer
 * @brief This class perform control tasks using multiple pulginlib plugins of
 * different algorithms
 *
 * Hosts FollowPath action server which receives path from action client and it
 * is then feed to controller which calculates and publishes twist velocities.
 */
```

Reply:
I was just trying to shorten it to just the `brief`, I'm not sure that the over context is needed except mentioning that this provides the FollowPath action server -- the rest is just mentioning the specifics of the plugins which other docs should cover.

also `pulginlib` -> `pluginlib`